packages/typography/src/Typography.story.tsx

```diff
@@ -24,6 +33,23 @@ storiesOf('Typography', module).add('Default', () => (
       <InlineCode className={displayBlock}>InlineCode</InlineCode>
       <Disclaimer className={displayBlock}>Disclaimer</Disclaimer>
       <Overline>Overline</Overline>
+      <Link
+        href={select(
+          'href',
+          [
+            'http://localhost:9001',
+            'https://mongodb.github.io/leafygreen-ui/?path=/story/*',
+          ],
+          'http://localhost:9001',
```

Comment:
Hmm... This is fun 😄 

<img width="2560" alt="Screen Shot 2020-07-20 at 2 08 30 PM" src="https://user-images.githubusercontent.com/25213299/87970962-9611b400-ca92-11ea-8ca1-39fa7cbfa0c0.png">

It's not infinite FYI, but some iFrame fun regardless!

Reply:
Haha! had to do that in order to demonstrate what a local link looks like..